core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalAppender.java

```diff
@@ -40,46 +39,38 @@ public class RaftJournalAppender implements Closeable {
   /** Hosting server for the appender. Used by default for appending log entries. */
   private final RaftServer mServer;
   /** local client ID, provided along with hosting server.  */
-  private ClientId mLocalClientId;
+  private final ClientId mLocalClientId;
   /** Remote raft client. */
   private final Supplier<RaftClient> mClientSupplier;
   private volatile RaftClient mClient;
-  /** Whether to use remote RaftClient for appending log entries. */
-  private boolean mEnableRemoteClient;
 
   /**
    * @param server the local raft server
    * @param clientSupplier a function for building a remote raft client
    * @param localClientId the client id for local requests
-   * @param configuration the server configuration
    */
   public RaftJournalAppender(RaftServer server, Supplier<RaftClient> clientSupplier,
-      ClientId localClientId, InstancedConfiguration configuration) {
+      ClientId localClientId) {
     mServer = server;
```

Comment:
If you don't expect any of the input to be null, consider use `mServer = requireNonNull(server, "server is null")`.

Reply:
fixed.